classes/requests/helpers/class-dintero-checkout-order.php

```diff
@@ -431,4 +432,20 @@ function( $value ) {
 			}
 		);
 	}
+
+	/**
+	 * Get the product's image URL.
+
+	 * @param  WC_Product|WC_Order_Item_Product $product Product.
+	 * @return string $image_url Product image URL. Empty string if no image is found.
+	 */
+	public function get_product_image_url( $product ) {
+		$image_url = '';
```

Comment:
It's better if the `image_url` is undefined if not set

Reply:
I've adjusted this function to return `false` which is the standard for when a request fails, and what the underlying functions that we use to retrieve the image also return.